app/src/stores/notification.ts

```diff
@@ -25,10 +36,18 @@ export const useNotificationStore = defineStore("notification", () => {
             });
         }, 100);
 
-        setTimeout(() => {
-            notifications.value = notifications.value.filter((n) => n.id != notificationId);
-        }, 2000);
+        if (notification.type == "toast") {
+            setTimeout(() => {
+                removeNotification(notificationId!);
+            }, 4000);
```

Comment:
4 seconds feels a bit long for me, just want to check if it was a conscious decision to double it?

Reply:
@belljohan3 If it was a conscious decision to do this it's fine, but please place a comment instead of just resolving, otherwise I don't know what the status is here 😅